src/main/webapp/js/user-page-loader.js

```diff
@@ -95,4 +96,21 @@ function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
   fetchMessages();
-}
\ No newline at end of file
+  fetchAboutMe();
+}
+
+/** Fetches about me data */
+function fetchAboutMe(){
+    const url = '/about?user=' + parameterUsername;
+    fetch(url).then((response) => {
+        return response.text();
```

Comment:
I'd recommend using `.json()` instead of `.text()`. This will make it easier to introduce more data in the response in the future. For example, let's say that right now the about me only consists of a short tagline. If you use JSON, you can have a response that looks something like:

```
{
   "tagline": "My tagline"
}
```

And then, in the future, if you want to add links to one's relevant social profiles as in:

```
{
   "tagline": "My tagline",
   "socialProfiles": [{
      "displayName": "Facebook",
      "url": "http://facebook.com/myprofile"
    }, {
      "displayName": "Twitter",
      "url": "http://twitter.com/myprofile"
    }]
}
```

... you can do so without breaking existing client code.

If, on the other hand, you put the tagline as just the raw text response, then when you add more content to the response, code that was assuming that the entire message is the tagline would be broken as a result of the update.